rebar.config

```diff
@@ -1,25 +1,39 @@
     {erl_opts, [debug_info,
-                warn_missing_spec,
+                %warn_missing_spec,
```

Comment:
What is the root cause behind disabling this flag?

Reply:
I guess I disabled this to prevent a wall of warnings every time i executed something. I will put it back in place.